src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/openapi/ClientsApiController.java

```diff
@@ -435,4 +432,18 @@ private ClientType getClientType(String encodedId) {
         Client result = clientConverter.convert(added);
         return createCreatedResponse("/api/clients/{id}", result, result.getId());
     }
+
+    @Override
+    @PreAuthorize("hasAuthority('SEND_CLIENT_REG_REQ')")
```

Comment:
According to Sonar this method is not tested. Most of ClientsApiController other methods are tested.

But this controller method does not contain anything nontrivial, it is probably correct to not test it. So just a FYI comment.

Reply:
Added controller tests